fixtures/packaging/browserify/dev/package.json

```diff
@@ -5,9 +5,10 @@
     "browserify": "^13.3.0",
     "react": "link:../../../../build/node_modules/react",
     "react-dom": "link:../../../../build/node_modules/react-dom",
-    "react-scheduler": "link:../../../../build/node_modules/react-scheduler"
+    "schedule": "link:../../../../build/node_modules/schedule"
   },
   "scripts": {
-    "build": "rm -f output.js && browserify ./input.js -o output.js"
+    "build": "rm -f output.js && browserify ./input.js -o output.js",
+    "prebuild": "rm -rf ./node_modules/react-dom/node_modules/schedule"
```

Comment:
Sketchy. :/ If we can't figure out how to make Yarn work with this, let's just remove dependencies completely, and then make this `cp -r ../../../../build/node_modules/* ./node_modules/`?